packages/cli/src/plugins/resource/plugin-standard-css.js

```diff
@@ -26,13 +26,23 @@ function bundleCss(body, url, compilation) {
       if ((type === 'String' || type === 'Url') && this.atrulePrelude && this.atrule.name === 'import') {
         const { value } = node;
 
-        if (value.indexOf('.') === 0 || value.indexOf('/node_modules') === 0) {
-          const resolvedUrl = value.startsWith('/node_modules')
-            ? new URL(`.${value}`, projectDirectory)
-            : new URL(value, url);
-          const importContents = fs.readFileSync(resolvedUrl, 'utf-8');
+        if (!value.startsWith('http')) {
+          if (value.indexOf('.') === 0 || value.indexOf('/node_modules') === 0) {
+            const resolvedUrl = value.startsWith('/node_modules')
```

Comment:
this will need to align with https://github.com/ProjectEvergreen/greenwood/pull/1341/files#diff-e20a60e7b88e17250746babe9a6f80b88def96d34e3901541b6ced7dd668b378R32